packages/manager/modules/iam/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@ovh-ux/manager-iam",
+  "version": "1.0.0",
+  "private": true,
+  "description": "IAM product.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/iam"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "peerDependencies": {
+    "@ovh-ux/ng-ovh-feature-flipping": "^1.0.7",
+    "@ovh-ux/ng-ovh-utils": "^14.0.19",
+    "@ovh-ux/ng-ui-router-breadcrumb": "^1.1.8",
+    "@ovh-ux/ui-kit": "^6.4.0",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "^1.7.5",
+    "angular-translate": "^2.18.1",
+    "lodash-es": "^4.17.15",
+    "oclazyload": "^1.1.0",
+    "ovh-ui-kit-bs": "^4.2.0"
+  },
+  "resolveAlias": {
```

Comment:
i think you should extend your webpack config file and setup your aliases here, just like in the doc : https://webpack.js.org/configuration/resolve/

Reply:
I don't think that's a good idea because we separate modules from apps, that leads to a lot of code repetition. The module needs the aliases, so it is his responsibility to declare them, I can't find a cleaner way of doing it ([others had the same idea](https://github.com/Rush/link-module-alias)). The apps that import the module have to resolve them for the module to work. They should not know about the details of the resolution